coriolis/tests/osmorphing/test_windows.py

```diff
@@ -0,0 +1,893 @@
+            self.detected_os_info)
+
+        self.assertFalse(result)
+
+    def test__get_worker_os_drive_path(self):
+        result = self.morphing_tools._get_worker_os_drive_path()
+
+        self.conn.exec_ps_command.assert_called_once_with(
+            "(Get-WmiObject Win32_OperatingSystem).SystemDrive")
+
+        self.assertEqual(result, self.conn.exec_ps_command.return_value)
+
+    @mock.patch.object(
+        windows.BaseWindowsMorphingTools, '_get_worker_os_drive_path'
+    )
+    def test__get_dism_path(self, mock_get_worker_os_drive_path):
+        result = self.morphing_tools._get_dism_path()
+
+        mock_get_worker_os_drive_path.assert_called_once_with()
+
+        self.assertEqual(
+            result,
+            "%s\\Windows\\System32\\dism.exe" %
+            mock_get_worker_os_drive_path.return_value
+        )
+
+    def test__get_sid(self):
+        result = self.morphing_tools._get_sid()
+
+        self.conn.exec_ps_command.assert_called_once_with(
+            "(New-Object System.Security.Principal.NTAccount($ENV:USERNAME))."
+            "Translate([System.Security.Principal.SecurityIdentifier]).Value"
+        )
+
+        self.assertEqual(result, self.conn.exec_ps_command.return_value)
+
+    def test__grant_permissions(self):
+        self.morphing_tools._grant_permissions(
+            mock.sentinel.path, mock.sentinel.user, perm="(OI)(CI)F")
+
+        self.conn.exec_command.assert_called_once_with(
+            'icacls.exe',
+            [mock.sentinel.path, '/grant', 'sentinel.user:(OI)(CI)F'])
+
+    def test__revoke_permissions(self):
+        self.morphing_tools._revoke_permissions(
+            mock.sentinel.path, mock.sentinel.user)
+
+        self.conn.exec_command.assert_called_once_with(
+            'icacls.exe',
+            [mock.sentinel.path, '/remove', mock.sentinel.user])
+
+    def test__load_registry_hive(self):
+        self.morphing_tools._load_registry_hive(
+            mock.sentinel.subkey, mock.sentinel.path)
+
+        self.conn.exec_command.assert_called_once_with(
+            'reg.exe',
+            ['load', mock.sentinel.subkey, mock.sentinel.path])
+
+    def test__unload_registry_hive(self):
+        self.morphing_tools._unload_registry_hive(mock.sentinel.subkey)
+
+        self.conn.exec_command.assert_called_once_with(
+            'reg.exe',
+            ['unload', mock.sentinel.subkey])
+
+    @mock.patch.object(windows.re, 'search')
+    def test__get_ps_fl_value(self, mock_search):
+        result = self.morphing_tools._get_ps_fl_value(
+            mock.sentinel.data, mock.sentinel.name)
+
+        mock_search.assert_called_once_with(
+            r"^%s\s*: (.*)$" % mock.sentinel.name, mock.sentinel.data,
+            windows.re.MULTILINE)
+
+        self.assertEqual(
+            result, mock_search.return_value.groups.return_value[0])
+
+    @mock.patch.object(windows.BaseWindowsMorphingTools, '_get_dism_path')
```

Comment:
You don't need to mock `_get_dism_path`, because you already mock `_get_worker_os_drive_path` which is used in the body of `_get_dism_path` method.